dpytools/slack/slack.py

```diff
@@ -1,14 +1,25 @@
+import os
+import logging
+from dpytools.http_clients.base import BaseHttpClient
 
 class SlackNotifier:
 
     def __init__(self):
```

Comment:
Code is good but I need to change something (sorry) - can we changed the pattern so the slack webhook url is coming from a positional argument rather than an environment variable.

It didn't occur to me before but there are scenarios where a single application may need to send messages to more than one channel.